ngraph/frontend/frontend_manager/include/frontend_manager/frontend_manager.hpp

```diff
@@ -119,4 +95,17 @@ namespace ngraph
 
     } // namespace frontend
 
+    template <>
+    class NGRAPH_API VariantWrapper<std::shared_ptr<std::istream>>
+        : public VariantImpl<std::shared_ptr<std::istream>>
+    {
+    public:
+        static constexpr VariantTypeInfo type_info{"Variant::std::shared_ptr<std::istream>", 0};
```

Comment:
@GlebKazantaev and @ilyachur, please provide your opinion: could we use standard C++ RTTI as a default supplier for type_info names to automate creation of VariantWrapper specializations? Looks like we don't obtain any real value by not doing that. I remember there were issues with standard RTTI on one of our supported platforms, do they still present?

Reply:
At the current moment we don't have issues with standard RTTI